nvidia_clara/model_types.py

```diff
@@ -139,12 +139,12 @@ def name(self, name: str = None):
         self._name = name
 
     @property
-    def tags(self) -> HashMap[str, str]:
+    def tags(self) -> Mapping[str, str]:
         """The set of tags / meta-data associated with this infrence model."""
         return self._tags
 
     @tags.setter
-    def tags(self, tags: HashMap[str, str] = None):
+    def tags(self, tags: Mapping[str, str] = None):

```

Comment:
`Mapping` vs `HashMap`, why the change?

Reply:
The python map/dictionary is a hash map, there is no actual HashMap class. Not sure how it was working before, but according to the Python official documentation using `Mapping` seems to be the proper convention.